### DISCUSSION TITLE
Bun v1.0.9

### DISCUSSION THREAD

**Discussion by Jarred-Sumner** [oven-sh/bun#6912] [Announcements]
To install Bun v1.0.9:
```bash
curl https://bun.sh/install | bash
# or you can use npm
# npm install -g bun
```

To upgrade to Bun v1.0.9:
```bash
bun upgrade
```

**[Read Bun v1.0.9's release notes on Bun's blog](https://bun.sh/blog/bun-v1.0.9)**

## What's Changed
* chore: build and publish canary windows builds by @paperdave in https://github.com/oven-sh/bun/pull/6834
* fix jsx key parse and visit order by @dylan-conway in https://github.com/oven-sh/bun/pull/6866
* more install tests by @dylan-conway in https://github.com/oven-sh/bun/pull/6867
* Fix edgecase from `eeb3f7b` by @Jarred-Sumner in https://github.com/oven-sh/bun/pull/6870
* fix: baseline+linux arm64 builds by @paperdave in https://github.com/oven-sh/bun/pull/6873
* Spawn ref count by @dylan-conway in https://github.com/oven-sh/bun/pull/6890
* Add more tests for Bun.spawn lifecycle and address edgecase by @Jarred-Sumner in https://github.com/oven-sh/bun/pull/6904

**Full Changelog**: https://github.com/oven-sh/bun/compare/bun-v1.0.8...bun-v1.0.9

<hr /><em>This discussion was created from the release <a href='https://github.com/oven-sh/bun/releases/tag/bun-v1.0.9'>Bun v1.0.9</a>.</em>

**Comment by jessicaadam0** [comment]
Hi 🤔
